site/content/deployment/openshift.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "OpenShift"
+weight = 310
++++
+
+All the Eclipse Hono&trade; components can be deployed on OpenShift, thanks to the resources YAML files that are provided through the repository.
+These files describe such components in terms of "deployments" and "services" in order to have the right pods running in the OpenShift cluster even communicate
```

Comment:
I do not get the meaning of the last sentence. I think there are some words missing?